README.md

```diff
@@ -141,12 +141,37 @@ build_windows_cuda.bat
 Build can take ~2 hours depending on the machine configuration.
 
 ### Build and test using docker
+
+### Prerequisites
+* Ensure virtualization is enabled in both the BIOS settings of your computer and the Windows virtualization feature
+* Install WSL 2 on your system
+* Use the command line to set WSL 2 as the default version by running the command: wsl --set-version 2
```

Comment:
Please show the command as "code" with windows compand prompt for clarity <>

Reply:
sure